lib/buildStartUpParams.js

```diff
@@ -19,11 +19,17 @@ function buildStartUpParams(configStore) {
     amorphicOptions.conflictMode = rootCfg.get('conflictMode') || amorphicOptions.conflictMode;
     amorphicOptions.sessionExpiration = rootCfg.get('sessionSeconds') * 1000;
     amorphicOptions.objectCacheExpiration = rootCfg.get('objectCacheSeconds') * 1000;
-    amorphicOptions.sessionSecret = rootCfg.get('sessionSecret');
     amorphicOptions.appList = rootCfg.get('applications');
     amorphicOptions.appStartList = rootCfg.get('application').split(';');
     amorphicOptions.mainApp = amorphicOptions.appStartList[0];
     amorphicOptions.port = rootCfg.get('port');
+
+    if (rootCfg.get('sessionSecret')) {
+        amorphicOptions.sessionSecret = rootCfg.get('sessionSecret');
+    } else {
+        amorphicOptions.sessionSecret = 'swat_team';
+        console.warn('WARNING you are starting amorphic with no sessionSecret. The default sessionSecret "swat_team" will be used.');
```

Comment:
should we use amorphic logger for this?

Reply:
I don't think amorphic logger is instantiated at this point. Which is why I used a `console.warn`

Reply:
There's an internal logging class you can use.  Check out `/lib/utils/logger.js`.